spec/support/quickbooks_online_helpers.rb

```diff
@@ -55,6 +177,9 @@ module QuickBooksOnlineHelpers
 
   def api_url(ledger_id: nil, resource:)
     ret = "https://sandbox-quickbooks.api.intuit.com/v3/company/realm_id/#{resource}"
+    if resource.to_s == 'preferences'
```

Comment:
Just a temporary hack untill we move some other resources and decide how we will do better resource to url mapping, coz we have few that are custom.